tests/models/test_simplemodel.py

```diff
@@ -16,9 +17,9 @@ def test_simplemodel(testdata_3days_path):
 
     assert len(paired_measurements) == len(filtered_measurement_pairs)
 
-    rarest_measurement_pair = \
-        sort_pairs_based_on_rarest_location(
+    _, rarest_measurement_pair = \
```

Comment:
dit gaat nog niet goed, get_pair_with_rarest_measurement_b geeft wel een tuple terug, maar die komt in zijn geheel in _ en niet in rarest_measurement_pair

Reply:
er lijkt bij mij een tuple1(tuple2(id, pair), None) uit te komen